guides/common/modules/proc_configuring-the-discovery-service.adoc

```diff
@@ -12,6 +12,9 @@ The Discovery service is enabled by default on {ProjectServer}.
 However, the default setting of the global templates is to boot from the local hard drive.
 To change the default setting, in the {ProjectWebUI}, navigate to *Administer* > *Settings*, and click the *Provisioning* tab.
 Locate the *Default PXE global template entry* row, and in the *Value* column, enter *discovery*.
+Then build the PXE default configuration.
+Navigate to *Hosts* > *Templates* > *Provisioning Templates* and click *Build PXE Default*.
+{Project} distributes the PXE configuration to any TFTP {SmartProxyServers}.
```

Comment:
```suggestion
ifdef::satellite[]
{Project} distributes the PXE configuration to any TFTP {SmartProxys}.
endif::[]
ifndef::satellite[]
{Project} distributes the PXE configuration to any TFTP {SmartProxys}.
endif::[]
```

Reply:
I can't see a difference between the two ifs. What is the point supposed to be?

Reply:
It's not my day yet!

I meant to say "TFTP Proxy" for non-Satellite.